restrict-merge-to-base-branch/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Restrict merging to the specified base branch only from a specified head branch
+description: A pull request can be merged only if the both the base and the head branch conform to the specified branch names.
+
+inputs:
+  head-ref-branch-name:
+    description: The name of the Head ref branch
+    required: true
+  base-ref-branch-name:
+    description: The name of the base ref branch
+    required: true

```

Comment:
From an end user point-of-view, do you think `source` and `destination` branch name would make more sense? Especially since this will be used on pull requests

Reply:
I was following github's convention of `base` and `head` but this works too! Thanks!